cmd/kube-scheduler/app/server_test.go

```diff
@@ -221,7 +220,7 @@ leaderElection:
 			name: "default config with an alpha feature enabled and an beta feature disabled",
 			flags: []string{
 				"--kubeconfig", configKubeconfig,
-				"--feature-gates=VolumeCapacityPriority=true,DefaultPodTopologySpread=false",
+				"--feature-gates=VolumeCapacityPriority=true,JobMutableNodeSchedulingDirectives=false",
```

Comment:
This is not a feature that affects kube-scheduler. Just get rid of the "and a beta feature disabled" part of the test. In this test, we should only care about features that affect the config.

Reply:
done